app/assets/stylesheets/content/_flash_messages.sass

```diff
@@ -42,6 +42,7 @@
 
 .flash
   display: none
+  padding: 15px 5px
```

Comment:
Let's separate the layers of this discussion:
1) The goal of a nicer styling for flash messages: I like it.
2) The technical implementation: I see room for improvements.

I believe that a change to a relative size is quite easy to do. As for why it should change, my answer is that is simply will. Because requirements change and we should build for that. A scenarios I can already see that make it likely is themes wanting to define their own font size.